packages/wonder-blocks-switch/src/themes/default.ts

```diff
@@ -0,0 +1,66 @@
+import {tokens} from "@khanacademy/wonder-blocks-theming";
+
+const theme = {
+    color: {
+        bg: {
+            primary: {
+                off: tokens.color.offBlack50,
+                disabledOff: tokens.color.offBlack32,
+                activeOff: tokens.color.offBlack64,
+                on: tokens.color.blue,
+                disabledOn: tokens.color.fadedBlue,
+                activeOn: tokens.color.activeBlue,
+            },
+            secondary: {
+                on: tokens.color.white,
+                off: tokens.color.white,
+            },
```

Comment:
**praise:** This is a great use of adding an extra level, as it helps to separate elements within the component! Also love the distinction between different states (so descriptive).